app/models/manageiq/providers/google/cloud_manager/event_parser.rb

```diff
@@ -0,0 +1,16 @@
+module ManageIQ::Providers::Google::CloudManager::EventParser
+  def self.event_to_hash(event, ems_id)
+    log_header = "ems_id: [#{ems_id}] " unless ems_id.nil?
```

Comment:
The `log_header`variable is not being used.  Looking at the `event_to_hash` method for other providers you might want to add a debug message here to follow the same pattern:

```ruby
_log.debug { "#{log_header}event: [#{event["event_type"]}]" }
```

Note: You can use `{ }` with the debug log message so the string only get interpolated if we are logging at the debug level.